content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -0,0 +1,57 @@
+---
+content_type: concept
+title: Perangkat untuk Memantau Sumber Daya
+---
+
+<!-- overview -->
+
+Untuk penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
+memahami bagaimana aplikasi berperilaku ketika aplikasi tersebut digelar. Kamu bisa memeriksa
```

Comment:
digelar doesn't seem familiar, how about adding parentheses (_deploy_) or change to "dijalankan"?

Reply:
Ok!